packages/ui/src/utils/ClipboardManager.ts

```diff
@@ -56,16 +55,15 @@ export class ClipboardManager {
           item.types.includes(ClipboardManager.KAOTO_MIME_TYPE)
         ) {
           const blob = await item.getType(ClipboardManager.KAOTO_MIME_TYPE);
-          const parsedContent = JSON.parse(await blob.text());
-          return updateIds(parsedContent);
+          return JSON.parse(await blob.text());
```

Comment:
@shivamG640 Just FYI, I needed to move `updateIds()` invocation to the caller side in order to get the original content accessible to the `paste-step.hook.tsx`. DataMapper step needs the original step ID to process the metadata.